contract_manager/scripts/upgrade_evm_entropy_contracts.ts

```diff
@@ -27,6 +28,17 @@ const parser = yargs(hideBin(process.argv))
     },
   });
 
+// Override these URLs to use a different RPC node for mainnet / testnet.
+// TODO: extract these RPCs to a config file (?)
+const RPCS: Record<string, string> = {
+  "mainnet-beta": "https://api.mainnet-beta.solana.com",
+  testnet: "https://api.testnet.solana.com",
+};
+1;
+function registry(cluster: PythCluster): string {
```

Comment:
Unrelated to this PR but I just noticed that you didn't add the new `registry` parameter into the JSDocs in `governance.ts`. It would be great to add it there.